packages/components/src/border-control/styles.ts

```diff
@@ -29,41 +29,15 @@ const focusBoxShadow = css`
 	box-shadow: inset 0 0 0 ${ CONFIG.borderWidth } ${ COLORS.ui.borderFocus };
 `;
 
-export const borderControl = css`
-	position: relative;
-`;
+export const borderControl = css``;
 
 export const innerWrapper = () => css`
-	flex: 1 0 40%;
-
-	/*
-	 * Needs more thought. Aim is to prevent the border for BorderBoxControl
-	 * showing through the control. Likely needs to take into account
-	 * light/dark themes etc.
-	 */
-	background: #fff;
-
-	/*
-	 * Forces the width control to fill available space given UnitControl
-	 * passes its className directly through to the input.
-	 */
-	${ UnitControlWrapper } {
-		flex: 1;
-		${ rtl( { marginLeft: -1 } )() }
-	}
-
 	&& ${ UnitSelect } {
-		/* Prevent default styles forcing heights larger than BorderControl */
+		/* Prevent unit select forcing min height larger than its UnitControl */
 		min-height: 0;
 		${ rtl(
-			{
-				borderRadius: '0 1px 1px 0',
-				marginRight: 0,
-			},
-			{
-				borderRadius: '1px 0 0 1px',
-				marginLeft: 0,
-			}
+			{ borderRadius: '0 2px 2px 0' },
+			{ borderRadius: '2px 0 0 2px' }
```

Comment:
I'm not sure I understand what these styles are for 🤔 Are they no longer needed...?

The `:focus` stuff too, should they be moved to `UnitControl` itself? (Not necessarily in this PR)

Reply:
I swear that when I was putting this PR together I still needed these styles or the unit select showed square corners. I must have been seeing things. After retesting the border-radius rules can be removed.

While testing the focus styles it looks like the focus styling in general for the `UnitControl` is a little broken e.g. wrapper's border overlays the unit selects focus border/box-shadow etc.

I've moved these focus styles to the `UnitControl` in a separate PR in the hope that might provide a better historical record if we need to revisit them in the future. See https://github.com/WordPress/gutenberg/pull/42383.

For this PR, I'll update it shortly to target the https://github.com/WordPress/gutenberg/pull/42383 branch as its base and simply remove the overrides from the `BorderControl`.